### PR TITLE
Add minimum requirements for MySQL notification

### PR DESCRIPTION
**Pull request by nitisht**
## Description
Added the minimum requirements for MySQL support in notification targets.

## Motivation and Context
This is important for users to know if they have proper setup to use Minio Mysql target.

## How Has This Been Tested?
Manually

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/bucket/notifications/README.md [hunk 1]
```diff
@@ -547,9 +547,9 @@ key                 |                      value
 <a name="MySQL"></a>
 ## Publish Minio events via MySQL
 
-Install MySQL from [here](https://dev.mysql.com/downloads/mysql/). For illustrative purposes, we have set the root password as `password` and created a database called `miniodb` to store the events.
+Install MySQL from [here](https://dev.mysql.com/downloads/mysql/). To publish Minio events, you'll need `JSON` support in MySQL, available on MySQL `5.7.8` and above. We tested this setup on MySQL `5.7.17`. For illustrative purposes, we have set the root password as `password` and created a database called `miniodb` to store the events.
```
Comment (donatello) [inline docs/bucket/notifications/README.md]: Can remove trailing space at end of line?

File: docs/bucket/notifications/README.md [hunk 2]
```diff
@@ -547,9 +547,9 @@ key                 |                      value
 <a name="MySQL"></a>
 ## Publish Minio events via MySQL
 
-Install MySQL from [here](https://dev.mysql.com/downloads/mysql/). For illustrative purposes, we have set the root password as `password` and created a database called `miniodb` to store the events.
+Install MySQL from [here](https://dev.mysql.com/downloads/mysql/). To publish Minio events, you'll need `JSON` support in MySQL, available on MySQL `5.7.8` and above. We tested this setup on MySQL `5.7.17`. For illustrative purposes, we have set the root password as `password` and created a database called `miniodb` to store the events. 
 
-This notification target supports two formats: _namespace_ and _access_.
+This notification target supports two formats: _namespace_ and _access_.
```
Comment (donatello) [inline docs/bucket/notifications/README.md]: Can remove trailing space at end of line?

**Review by donatello — CHANGES_REQUESTED** [review]

File: docs/bucket/notifications/README.md (hunk 1 above)
Comment (deekoder) [inline docs/bucket/notifications/README.md]: How does the user get JSON support? Just upgrade MySQL?

**Review by deekoder — CHANGES_REQUESTED** [review]

File: docs/bucket/notifications/README.md (hunk 1 above)
Comment (donatello) [reply docs/bucket/notifications/README.md]: Yes, JSON data-type support in MySQL is available from a particular version onwards.

Comment (deekoder) [reply docs/bucket/notifications/README.md]: ok. i approve pending the space removals.

**Review by deekoder — DISMISSED** [review]
Pending : space removals.

**Review by donatello — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**